src/MacroExpander.js

```diff
@@ -266,16 +272,24 @@ export default class MacroExpander implements MacroContextInterface {
      * tokens, or return `undefined` if no such macro is defined.
      */
     expandMacro(name: string): Token[] | void {
-        if (!this.macros.get(name)) {
-            return undefined;
-        }
+        return this.macros.has(name)
+            ? this.expandTokens([new Token(name)]) : undefined;
+    }
+
+    /**
+     * Fully expand the given token stream and return the resulting list of tokens
+     */
+    expandTokens(tokens: Token[]): Token[] {
         const output = [];
         const oldStackLength = this.stack.length;
-        this.pushToken(new Token(name));
+        this.pushTokens(tokens);
         while (this.stack.length > oldStackLength) {
-            const expanded = this.expandOnce();
+            const expanded = this.expandOnce(true);
             // expandOnce returns Token if and only if it's fully expanded.
             if (expanded instanceof Token) {
```

Comment:
So whenever `expandOnce()` returns an array it's not considered fully expanded?

Reply:
Yes.